include/openssl/rand_drbg.h

```diff
@@ -106,16 +106,16 @@ int RAND_DRBG_set_reseed_interval(RAND_DRBG *drbg, unsigned int interval);
 int RAND_DRBG_set_reseed_time_interval(RAND_DRBG *drbg, time_t interval);
 
 int RAND_DRBG_set_reseed_defaults(
-                                  unsigned int master_reseed_interval,
-                                  unsigned int slave_reseed_interval,
-                                  time_t master_reseed_time_interval,
-                                  time_t slave_reseed_time_interval
+                                  unsigned int parent_reseed_interval,
+                                  unsigned int child_reseed_interval,
+                                  time_t parent_reseed_time_interval,
+                                  time_t child_reseed_time_interval
                                   );
 
-RAND_DRBG *OPENSSL_CTX_get0_master_drbg(OPENSSL_CTX *ctx);
+RAND_DRBG *OPENSSL_CTX_get0_parent_drbg(OPENSSL_CTX *ctx);
 RAND_DRBG *OPENSSL_CTX_get0_public_drbg(OPENSSL_CTX *ctx);
 RAND_DRBG *OPENSSL_CTX_get0_private_drbg(OPENSSL_CTX *ctx);
-RAND_DRBG *RAND_DRBG_get0_master(void);
+RAND_DRBG *RAND_DRBG_get0_parent(void);
```

Comment:
This one is troublesome, as it exists in 1.1.1.

Reply:
Simple enough to keep the old name in addition, or deprecate it; let me know what you prefer.

Reply:
You'll have to keep the old name in addition no matter what.  If it were me, I would deprecate it, but from what I gather, that may be a bit too rad, so just add a backward compat macro.

Reply:
I will add a macro in the header file once the hold is removed.  I hope that will happen soon.

Reply:
Is the function part of the stable ABI? If yes then a macro is not sufficient.

Reply:
> Is the function part of the stable ABI? If yes then a macro is not sufficient.

Sorry, I missed Rich's reply https://github.com/openssl/openssl/pull/12089#discussion_r437389944